pkg/tidydns/tidydns.go

```diff
@@ -63,9 +68,32 @@ type RecordInfo struct {
 	Location    LocationID
 }
 
+type UserInfo struct {
+	ModifiedBy        string
+	Description       string
+	ModifiedDate      time.Time
+	Username          string
+	AuthGroup         AuthGroup
+	Name              string
+	PasswdChangedDate time.Time
+	Id                UserID
+	Groups            []UserInfoGroup
+}
+
+type UserInfoGroup struct {
+	GroupName   string
+	Name        string
+	Notes       *string
+	Id          int
+	Description *string
+}
```

Comment:
Here we should just use the `UserGroup` type since they are the same right?

Reply:
See reply on earlier comment

Reply:
Refactored in 9ab4bf49ad1f8ba99df183ded8c1d6f27a094fa3